### ISSUE TITLE
Number Pad

### ISSUE THREAD

**Issue by westendal** [tekezo/Karabiner-Elements#1203]
Just upgraded to OS 10.13.2

**Comment by westendal** [comment]
Just upgraded to os 10.13.2.  Formerly, I was able to use Karibiner to instruct my Mac to regard the keypad on the Dell USB kbd I use as cursor-control buttons . With the new OS (and the new KE), I can't figure out how to recover that functionality. The keypad seems to function fine as a number-selection device; I'd like, tho, to be able to use it for cursor/page control. ANy ideas?

**Comment by westendal** [comment]
Figured it out!

**Comment by davebest** [comment]
I've got the same problem with a compact Logitech keyboard.  How did you solve it?

**Comment by westendal** [comment]
In the original Karibiner pgm, there was an opportunity to convert the keypad in just one step.
With Elements, I had to convert each key, one at a time.

Open Elements, go to Preferences, and see the drop-down menu that allows you to select the kbd key you want to work on. Select it from the list.
Then go to the drop-down menu on the same line, but to the right, and select the key’s new meaning.

Since there are 17 keys on the key pad, on mine,at any rate, I had to specify 17 conversions.
But, having done that, it seems to be working just fine.

Good luck!
